ns1/resource_apikey_test.go

```diff
@@ -246,6 +248,7 @@ func testAccAPIKeyPermissionsOnTeam(rString string) string {
 	return fmt.Sprintf(`resource "ns1_team" "t" {
   name = "terraform acc test team %s"
   account_manage_account_settings = true
+  account_manage_ip_whitelist = true
```

Comment:
Is this necessary?  It doesn't look like you're checking this param in the test that uses this config.

Reply:
it used on the next step after removing the team permissions should stay in apikey